backend/src/gpml/db/review.sql

```diff
@@ -4,7 +4,16 @@ SELECT * FROM review ORDER BY id;
 
 -- :name reviews-by-reviewer-id :? :*
 -- :doc Get reviews for reviewer
-SELECT * FROM review
+SELECT r.*, (CASE
```

Comment:
amazing! I'm learning a lot of postgres with this review 😄 !

Reply:
Haha, I'm not sure if that's a good thing. :)